default.py

```diff
@@ -11,6 +11,11 @@
 from operator import itemgetter
 
 import requests
+import requests.packages.urllib3
+
+import cookielib
```

Comment:
This may be a stupid question as I have never worked with cookies: Is there a reason you import cookielib on top of requests? Requests claims it is able to handle cookies (http://docs.python-requests.org/en/latest/user/quickstart/#cookies) and from my point of view it would be better to limit the number of additional imports.